connectors/cybereason/configurations/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "connection": {
+        "type": {
+            "type": "connectorType",
+            "displayName": "Cybereason"
+        }
+    },
+    "configuration": {
+        "auth": {
+            "type": "fields",
+            "cybereason_url": {
+                "type": "text"
+            },
+            "username": {
+                "type": "text"
+            },
+            "password": {
+                "type": "password"
+            },
+            "malop_retention_period": {

```

Comment:
What is the function of this parameter? Can we just use the schedule vale given by user?